cmd/fetch_test.go

```diff
@@ -17,6 +17,12 @@ import (
 )
 
 func TestFetchCommand(t *testing.T) {
+	t.Skip(`This currently won't work b/c commands can't be executed while the "temp registry" is running.
```

Comment:
Ran this locally and it's passing?

Reply:
if you run it like 10x, eventually you get something like this:

```
$ go test --run TestFetchCommand --count 10
2020-06-15T08:52:23.778-0400	INFO	qriapi	api/api.go:251	running in `remote` mode
2020-06-15T08:52:23.983-0400	INFO	qriapi	api/middleware.go:15	GET /remote/refs 2020-06-15 08:52:23.983783 -0400 EDT m=+7.919951329
2020-06-15T08:52:23.984-0400	INFO	qriapi	api/middleware.go:15	GET /remote/logsync 2020-06-15 08:52:23.984282 -0400 EDT m=+7.920450275
2020-06-15T08:52:24.126-0400	INFO	qriapi	api/middleware.go:15	GET /remote/refs 2020-06-15 08:52:24.126154 -0400 EDT m=+8.062319827
2020-06-15T08:52:24.126-0400	INFO	qriapi	api/middleware.go:15	GET /remote/logsync 2020-06-15 08:52:24.126606 -0400 EDT m=+8.062772636
2020-06-15T08:52:24.161-0400	ERROR	flatfs	go-ds-flatfs@v0.4.4/flatfs.go:993	could not store final value of disk usage to file, future estimates may be inaccurate
2020-06-15T08:52:24.162-0400	ERROR	flatfs	go-ds-flatfs@v0.4.4/flatfs.go:993	could not store final value of disk usage to file, future estimates may be inaccurate
2020-06-15T08:52:25.055-0400	INFO	qrip2p	p2p/bootstrap.go:35	error connecting to host: swarm closed
2020-06-15T08:52:25.818-0400	INFO	qriapi	api/api.go:251	running in `remote` mode
2020-06-15T08:52:26.162-0400	INFO	qriapi	api/middleware.go:15	GET /remote/refs 2020-06-15 08:52:26.162315 -0400 EDT m=+10.098455426
2020-06-15T08:52:26.163-0400	INFO	qriapi	api/middleware.go:15	GET /remote/logsync 2020-06-15 08:52:26.16357 -0400 EDT m=+10.099710974
--- FAIL: TestFetchCommand (2.23s)
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0xe0 pc=0x52aee7d]

goroutine 12674 [running]:
testing.tRunner.func1.1(0x5619960, 0x6c4a130)
	/usr/local/Cellar/go/1.14.3/libexec/src/testing/testing.go:940 +0x2f5
testing.tRunner.func1(0xc0004a26c0)
	/usr/local/Cellar/go/1.14.3/libexec/src/testing/testing.go:943 +0x3f9
panic(0x5619960, 0x6c4a130)
	/usr/local/Cellar/go/1.14.3/libexec/src/runtime/panic.go:969 +0x166
github.com/qri-io/qri/lib.(*LogMethods).Log(0xc0008fdbe0, 0xc0007bc680, 0xc000a5b540, 0x58a9546, 0x1)
	/Users/b5/qri/qri/lib/log.go:45 +0x3d
github.com/qri-io/qri/api.(*LogHandlers).logHandler(0xc0008fdbe0, 0x5d513c0, 0xc0004838c0, 0xc000ae2a00)
	/Users/b5/qri/qri/api/log.go:76 +0x43d
github.com/qri-io/qri/api.(*LogHandlers).LogHandler(0xc0008fdbe0, 0x5d513c0, 0xc0004838c0, 0xc000ae2a00)
	/Users/b5/qri/qri/api/log.go:34 +0x8e
github.com/qri-io/qri/cmd.APICall(0x58a9f3a, 0x3, 0x58dcf29, 0x1b, 0xc000151da8, 0xc000151d58, 0x0, 0x0, 0x0)
	/Users/b5/qri/qri/cmd/fetch_test.go:215 +0x458
github.com/qri-io/qri/cmd.TestFetchCommand(0xc0004a26c0)
	/Users/b5/qri/qri/cmd/fetch_test.go:179 +0xf01
testing.tRunner(0xc0004a26c0, 0x5a8b398)
	/usr/local/Cellar/go/1.14.3/libexec/src/testing/testing.go:991 +0xdc
created by testing.(*T).Run
	/usr/local/Cellar/go/1.14.3/libexec/src/testing/testing.go:1042 +0x357
exit status 2
FAIL	github.com/qri-io/qri/cmd	11.488s
```